contracts/extension/plugin/RouterImmutable.sol

```diff
@@ -18,4 +18,7 @@ contract RouterImmutable is Router {
     function _canSetPlugin() internal pure override returns (bool) {
         return false;
     }
+
+    /// @dev Returns whether a plugin is a safe, authorized plugin.
+    function _isAuthorizedPlugin(bytes4, address) internal view override returns (bool) {}
```

Comment:
make it explicitely `return false` here, reads more clearly

Reply:
This function is now deleted.

Reply:
did you push?